src/net.h

```diff
@@ -677,6 +677,7 @@ class CNode
 
 
     uint64_t nLocalHostNonce;
+    // What local services you advertised on connection to this node
```

Comment:
@rebroad afaik it is to keep track of what services you advertised to each peer. It could change over the lifetime of the connection.